data/category-payment-ir

```diff
@@ -5,10 +5,12 @@ asanpardakht.net
 behpardakht.com
 iranicard.ir
 mydigipay.com
+pay.ir
 payping.ir
 pec.ir
 pep.co.ir
 sadadpsp.ir
 sep.ir
 shaparak.ir
-zarinpal.ir
```

Comment:
BTW, why is this domain been deleted?

Reply:
the main domain is zarinpal.com

Reply:
zarinpal does not use this domain (zarinpal.ir) anymore and the host is not set on it

https://who.is/dns/zarinpal.ir

![photo_2023-06-07_16-13-18](https://github.com/v2fly/domain-list-community/assets/20362833/b86f66c5-313e-4747-a83b-107c9ae049c7)

Reply:
The DNS record does not matter here, this domain is still active.
You said zarinpal does not use this domain (zarinpal.ir) anymore, is there any source mentioned about that?

Reply:
> The DNS record does not matter here, this domain is still active. You said zarinpal does not use this domain (zarinpal.ir) anymore, is there any source mentioned about that?

I didn't see Zarinpal using the .ir domain anymore, but now if you think it should be in the list, I will add it